test/models/work_test.rb

```diff
@@ -0,0 +1,137 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { works(:harrypotter)  }
+
+  it "must be valid" do
+    expect(work).must_be :valid?
+  end
+
+  it 'has required fields' do
+    fields = [:title, :creator, :description, :publication_year, :category]
+
+    fields.each do |field|
+      expect(work).must_respond_to field
+    end
+  end
+
+  describe 'Relationships' do
+    it 'can have many votes' do
+      # Arrange (done with let)
+      work = works(:harrypotter)
+      vote1 = votes(:voteone)
+      # Act
+      vote = work.votes
+
+      # Assert
+      expect(vote).must_be_instance_of Vote
```

Comment:
because `vote` here is actually a collection of  votes, this test failed.  If you wanted to test the type you could loop through the collection and check that each instance is a `Vote`.  

Also since a `Work` has many votes, it doesn't have a foreign key in it.  

Instead `expect(vote.work_id).must_equal work.id`